hr_contract_cae/models/hr_contract.py

```diff
@@ -86,6 +90,11 @@ class Contract(models.Model):
     reason = fields.Char(string="Reason for recourse to contract", copy=False)
 
     duration = fields.Integer(string="Duration", default=6)
+    date_initial_start = fields.Date(
```

Comment:
`initial_start_date` is better english

Reply:
This was for symmetry with `date_start` and `date_end` in `hr.contract`